Upstream-Status: Inappropriate [Backport]

Make 3.82, as shipped with Fedora 14, fixes some holes in the parser which in
turn breaks behaviour of some Makefiles. Most notably eglibc's.

http://www.mail-archive.com/bug-make@gnu.org/msg06220.html

Fix back-ported from glibc/eglibc revision control.

JL 13/10/10

Index: libc/manual/Makefile
===================================================================
--- libc.orig/manual/Makefile
+++ libc/manual/Makefile
@@ -243,7 +243,10 @@ ifdef objpfx
 .PHONY: stubs
 stubs: $(objpfx)stubs
 endif
-$(objpfx)stubs ../po/manual.pot $(objpfx)stamp%:
+$(objpfx)stubs ../po/manual.pot:
+	$(make-target-directory)
+	touch $@
+$(objpfx)stamp%:
 	$(make-target-directory)
 	touch $@
 
